www/pages/guides/netlify-deploy.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Deploy on Netlify'
+menu: side
+linkheadings: 3
+index: 1
+---
+
+## Deploying your site on Netlify
+
+Requires an account with [Netlify](https://www.netlify.com) linked to your GitHub account. If you need to sign up, the process is simple as is the deployment.
+
+After you have created your Greenwood project and pushed your code to a GitHub repo...
+
+- Log in to your Netlify account and proceed to the dashboard.
```

Comment:
Sorry, same as with the other one, should have caught this before, re: using an ordered / numbered list would be more appropriate?